airbyte-integrations/connectors/source-mailchimp/source_mailchimp/schemas/email_activity.json

```diff
@@ -49,7 +49,7 @@
     "url": {
       "type": ["string", "null"],
       "title": "The IP address.",
```

Comment:
@dlackty can you correct the title of this field too?

Reply:
@marcosmarxm Updated & rebased!